index.js

```diff
@@ -1,33 +1,40 @@
+/* eslint-disable max-len */
```

Comment:
80 characters is quite short, I think that rule is too strict. Instead of disabling the rule entirely, maybe override the rule in .eslintrc.js to allow up to 100 chars

Reply:
Done